docs/tutorial/convenient-transactions.txt

```diff
@@ -0,0 +1,64 @@
+===============================
+Convenient API for Transactions
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+With the Convenient API for Transactinos, users don't have to implement
+any transaction management logic themselves. The ``MongoDB\\with_transaction()``
```

Comment:
I'd like to link this to the reference for the function, but I'm not sure if using `:phpmethod:` would work since the function reference is stored in a different folder.

Reply:
This actually works:

```
:phpmethod:`MongoDB\\with_transaction()`
```

It has nothing to do with folder organization. Rather, this picks up on `.. phpmethod:: MongoDB\\with_transaction()` in the reference files.